src/app/public/modules/modal/modal-scroll-shadow.directive.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  Directive,
+  ElementRef,
+  EventEmitter,
+  HostListener,
+  OnDestroy,
+  OnInit,
+  Output
+} from '@angular/core';
+
+import {
+  Subject
+} from 'rxjs';
+
+import {
+  takeUntil
+} from 'rxjs/operators';
+
+import {
+  MutationObserverService
+} from '@skyux/core';
+
+import {
+  SkyTheme,
+  SkyThemeService
+} from '@skyux/theme';
+
+import {
+  SkyModalScrollShadowEventArgs
+} from './modal-scroll-shadow-event-args';
+
+/**
+ * @internal
+ */
+@Directive({
+  selector: '[skyModalScrollShadow]'
```

Comment:
This directive is 🍌 . Love the effect.

Would you mind putting a description here so future devs can understand its purpose?

Reply:
Fixed.